service/orchestrator/toe.go

```diff
@@ -200,16 +200,16 @@ func (svc *Service) ListControlMonitoringStatus(ctx context.Context, req *orches
 		return nil, service.ErrPermissionDenied
```

Comment:
Shouldn't you return a status error?

Reply:
Ah lol! `var ErrPermissionDenied = status.Errorf(codes.PermissionDenied, "access denied")`